ad7616/system_bd.tcl

```diff
@@ -50,20 +50,18 @@ adi_project_files [list \
 
 source ../../projects/ad7616_sdz/common/ad7616_bd.tcl
 
-if {$SI_OR_PI == 0} {
+if {$SER_PAR_N == 1} {
 
   create_bd_port -dir O spi_clk
   create_bd_port -dir O irq
 
   ad_connect spi_clk sys_cpu_clk
   ad_connect irq spi_ad7616/irq
 
-} elseif {$SI_OR_PI == 1} {
+} else {
 
   create_bd_port -dir O sys_clk
-  create_bd_port -dir O irq
```

Comment:
In this case, it would be a little bit confusing because of the usage of the spi_clk in the project (can be a single clock in both environments (SI/PI), named sys_clk, because of the fact that the spi_clk has a frequency of 100 MHz, same as the sys_cpu_clk, that is connected to pl_clk0 output from the Zynq7000SoC IP).

Reply:
Will keep spi_clk for consistency, regardless the frequency.